pipeline/lib/utils.groovy

```diff
@@ -0,0 +1,38 @@
+#!groovy
+
+import com.cloudbees.plugins.credentials.CredentialsMatchers
+import com.cloudbees.plugins.credentials.CredentialsScope
+import com.cloudbees.plugins.credentials.SystemCredentialsProvider
+import com.cloudbees.plugins.credentials.domains.Domain
+import hudson.util.Secret
+import org.jenkinsci.plugins.plaincredentials.impl.StringCredentialsImpl
+
+
+def addSecretString(Secret secret, String description) {
+
+  credentialsId = java.util.UUID.randomUUID().toString()
+
+  credentials = new StringCredentialsImpl(
+    CredentialsScope.GLOBAL, credentialsId, description, secret)
+
+  getCredentialsStore().addCredentials(Domain.global(), credentials)
+
+  return credentialsId
+}
+
+def removeCredentials(String credentialsId) {
+  domain = Domain.global()
+
+  credentials = CredentialsMatchers.filter(
+    getCredentialsStore().getCredentials(domain),
+    CredentialsMatchers.withId(credentialsId))
+
+  getCredentialsStore().removeCredentials(domain, credentials.get(0))
```

Comment:
What is returned by `credentials.get(0)`? Maybe use a intermediary variable to describe it.